lib/features/chat/repository/chat_remote_repository.dart

```diff
@@ -102,26 +102,37 @@ class ChatRemoteRepository {
 
       // Iterate through chats and map users
       for (var chat in chatData) {
-        final chatID = chat['id'];
+        final chatID = chat['chat']['id'];
+
+        final members = (chat['chat']['members'] as List)
+            .map((member) => member['user'] as String)
+            .toList();
```

Comment:
why don't we add a condition that he is a normal member: 'member'
as we made a list for admins and a list for creators